pandas/io/gbq.py

```diff
@@ -691,12 +693,17 @@ def read_gbq(query, project_id=None, index_col=None, col_order=None,
 
         .. versionadded:: 0.18.1
 
-    dialect : {'legacy', 'standard'}, default 'legacy'
+    dialect : {'legacy', 'standard', None}, default None
         'legacy' : Use BigQuery's legacy SQL dialect.
         'standard' : Use BigQuery's standard SQL (beta), which is
         compliant with the SQL 2011 standard. For more information
         see `BigQuery SQL Reference
         <https://cloud.google.com/bigquery/sql-reference/>`__
+        None : Don't explicitly specify query dialect. This enables
```

Comment:
oh ok then. please put up a nice note in API changes section of whatsnew explaining this then.